common/general.d.ts

```diff
@@ -15,20 +16,32 @@ declare const enum SectionName {
 declare const enum CustomNpcKeys {
     SlacksMudGolem = "npc_mud_golem_slacks",
     SunsFanMudGolem = "npc_mud_golem_sunsfan",
+    GodzMudGolem = "npc_mud_golem_godz",
     Mirana = "npc_dota_hero_mirana",
     DireRangedCreep = "npc_dota_tutorial_dire_ranged_creep",
     DireMeleeCreep = "npc_dota_tutorial_dire_melee_creep",
     RadiantRangedCreep = "npc_dota_tutorial_radiant_ranged_creep",
     RadiantMeleeCreep = "npc_dota_tutorial_radiant_melee_creep",
     Riki = "npc_dota_hero_riki",
-    GodzMudGolem = "npc_mud_golem_godz",
     TargetDummy = "npc_dota_tutorial_target_dummy",
 }
 
 declare const enum CustomAbilityKeys {
     CustomMiranaArrow = "custom_mirana_arrow",
 }
 
+declare const enum chapter2SpecificKeys {
```

Comment:
Do these types need to be in general or even common if they are chapter 2 specific? If they are not sent to panorama they should not be in common. Also this name is breaking our naming convention, should start with capital C.

Reply:
Moved to the chapter directory without `declare const` and imported on the chapter 2 portions of it.